modules/carto/src/layers/utils.ts

```diff
@@ -19,9 +20,10 @@ export function mergeBoundaryData(geometry: VectorTile, properties: PropertiesTi
   const mapping = {};
   for (const {geoid, ...rest} of properties.properties) {
     if (geoid in mapping) {
-      throw new Error(`Duplicate geoid key in mapping: ${geoid}`);
+      log.warn(`Duplicate geoid key in boundary mapping, using first occurance`)();
```

Comment:
```suggestion
      log.warn(`Duplicate geoid [${geoid}] key in boundary mapping, using first occurance`)();
```

Reply:
I'm deliberately not doing that as it will then potentially log a huge number of messages, but I can add it if you think that is OK

Reply:
You're right